src/StandardModel.cc

```diff
@@ -217,9 +226,10 @@ double AlphaStrong::alphaS( double scale2) {
     }
     double logScale    = log(scale2/Lambda2);
     double loglogScale = log(logScale);
-    valueNow = 12. * M_PI / (b0 * logScale)
-      * ( 1. - b1 * loglogScale / logScale
-        + pow2(b1 / logScale) * (pow2(loglogScale - 0.5) + b2 - 1.25) );
+    double correction  = 1. - b1 * loglogScale / logScale;
+    if (order == 3) correction
```

Comment:
Hi @mseidel42. Wasn't this correction previously always applied? So shouldn't one add the !fixRunning ?